_pkgdown.yml

```
@@ -23,4 +23,6 @@ reference:
   - adjust_predictions_custom
 - title: "Utilities"
   contents:
-  - tidy.tailor
+  - fit.tailor
+  - predict.tailor
+  - tidy.tailor
```

```suggestion
  - tidy.tailor
  
```

Just adding a trailing newline.

FWIW, I had to change a setting in Positron to always add one!